src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OppgaveClient.kt

```diff
@@ -60,7 +60,8 @@ class OppgaveClient(
     fun finnMapper(finnMappeRequest: FinnMappeRequest): FinnMappeResponseDto {
         val response = getForEntity<Ressurs<FinnMappeResponseDto>>(
             UriComponentsBuilder.fromUri(URI.create("$oppgaveUrl/mappe/sok"))
-                .queryParams(finnMappeRequest.toQueryParams())
+                .queryParam("enhetsnr", finnMappeRequest.enhetsnr)
```

Comment:
Nå husker jeg ikke i farta hvordan dette var, men her kan vi risikere å gå på en smell, eller? Hva skjer hvis enhetsnr er "null" ? Sender vi `?enhetsnr=null` ? i såfall kan det bli en smell dersom det kommer frem til oppgave-apiet - for da får du ingen treff... :)

Reply:
Blir kalt med dette som parameter, så eg trur det skal vera greitt:
`val finnMappeRequest = FinnMappeRequest(
                listOf(),
                oppgave.tildeltEnhetsnr ?: error("Fikk ikke tildelt enhetsnummer for oppgave med id: $oppgaveId"),
                null,
                1000
            )`

Reply:
Det som derimot _kanskje_ blir ein smell er det Johan peiker på i tilsvarande PR på personhendelse: https://github.com/navikt/familie-ef-personhendelse/pull/299#discussion_r1034752647

Reply:
Gjorde det samme som OC endret i personhendelse